Cargo.toml

```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/martinjrobins/diffsol"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["nalgebra", "faer"]
+default = ["nalgebra", "faer", "sundials"]
```

Comment:
```suggestion
default = ["nalgebra", "faer"]
```

Reply:
I think we want to keep the default features as pure rust, sundials wraps a c library that users might need to compile if they don't already have it on their system.